dallinger/data.py

```diff
@@ -281,12 +290,23 @@ def export_db_uri(id, db_uri, local, scrub_pii):
                 f" - bucket name: {bucket.name}\n"
                 f" - S3 console URL: {s3_console_url}"
             )
-        except AttributeError:
-            raise S3BucketUnavailable("Could not find an S3 bucket!")
+        else:
+            print("✘ Could not find an S3 bucket!")
 
     return path_to_data
 
 
+def aws_access_keys_present(config):
```

Comment:
This function should return a boolean, I think. I don't think it should print a warning message like it currently does. You want to write instead something like 

```py
if not aws_access_keys_present(config):
    logger.warn("AWS credentials not present, skipping export to S3.")
```

Reply:
Done.

Reply:
Unfortunately, no matter which logging level I use, nothing gets printed on the console. So I reverted to `print`.